migrations/postgres-integration/0001_initial.up.sql

```diff
@@ -0,0 +1,117 @@
+CREATE EXTENSION IF NOT EXISTS hstore;

```

Comment:
I think it is better to remove this line. On purpose, it is not included in the other migrations. There is no guarantee that the user used for this database has permissions to add extensions. E.g. one might have a managed database. Using the admin credentials one would create the database add the extensions, and then use a dedicated users for ChirpStack.

Reply:
Got it 👍